pkgs/development/tools/build-managers/meson/default.nix

```
@@ -80,6 +82,8 @@ python3.pkgs.buildPythonApplication rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
+  propagatedBuildInputs = [ ninja ];
```

Should be runnable on buildPlatform
```suggestion
  propagatedNativeBuildInputs = [ ninja ];
```